
From: Jesper Juhl <juhl-lkml@dif.dk>

drivers/video/tdfxfb.c:1104: warning: initialization discards qualifiers from pointer target type

Signed-off-by: Jesper Juhl <juhl-lkml@dif.dk>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/drivers/video/tdfxfb.c |    2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)

diff -puN drivers/video/tdfxfb.c~tdfxb-warning-fix drivers/video/tdfxfb.c
--- 25/drivers/video/tdfxfb.c~tdfxb-warning-fix	2004-06-20 03:46:50.254670024 -0700
+++ 25-akpm/drivers/video/tdfxfb.c	2004-06-20 03:46:50.259669264 -0700
@@ -1101,7 +1101,7 @@ static int tdfxfb_cursor(struct fb_info 
 		 */
 		u8 *cursorbase = (u8 *) info->cursor.image.data;
 		char *bitmap = (char *)cursor->image.data;
-		char *mask = cursor->mask;
+		const char *mask = cursor->mask;
 		int i, j, k, h = 0;
 
 		for (i = 0; i < 64; i++) {
_
